src/inspector_js_api.cc

```diff
@@ -87,18 +87,26 @@ class JSBindingsConnection : public AsyncWrap {
     JSBindingsConnection* connection_;
   };
 
-  JSBindingsConnection(Environment* env,
+  JSBindingsConnection(bool is_sync,
+                       Environment* env,
```

Comment:
Tiny nit, obviously not super important: Can you make `is_sync` the last parameter instead of the first? The signatures for all `BaseObject` subclasses take `env` and `wrap` as the first arguments, to that would make it a bit more consistent